bin/bundle

```diff
@@ -1,3 +1,109 @@
 #!/usr/bin/env ruby
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
-load Gem.bin_path('bundler', 'bundle')
+# frozen_string_literal: true
+
```

Comment:
not sure this should by part of this PR?

Reply:
moved out to its out PR  #7633